app/workers/callbacks/http_callback.rb

```diff
@@ -10,7 +10,11 @@ def http_callback(web_hook)
   alias_method :https_callback, :http_callback
 
   def http_execute(uri, data, options={})
-    connection = Excon.new(uri.to_s, ssl_verify_peer: ENV['SSL_VERIFY_PEER'] )
+    connection = Excon.new(
+      uri.to_s,
+      ssl_verify_peer: ENV['SSL_VERIFY_PEER'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.